src/Sulu/Bundle/MediaBundle/Resources/translations/admin.de.json

```diff
@@ -32,6 +32,7 @@
     "sulu_media.mime_type": "MIME-Typ",
     "sulu_media.license": "Lizenz",
     "sulu_media.copyright": "Copyright Informationen",
+    "sulu_media.credits": "Credits",
```

Comment:
That's not the german translation :wink:

Reply:
In Sulu 1.6 it was called `Bildnachweis`, but the copyright is in 1.6 also translated to `Urheberrechtsinformationen ` should we change this translation as well ?

Reply:
To what should we change it? If you have no better idea I would just take the last value from 1.6.